subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1236,31 +1244,126 @@ ssize_t ztls_recv(int sock, void *buf, size_t max_len, int flags)
 	return ztls_recvfrom(sock, buf, max_len, flags, NULL, 0);
 }
 
+static ssize_t sendto_tls(struct net_context *context, const void *buf,
```

Comment:
Why is this called "sendto", if it doesn't accept dest addr param?

Reply:
My intention was to keep the naming uniform, as this static function is part of the `sendto` function, but sure I can rename it. Same with others.

Reply:
I see, but "sendto" and "recvfrom" socket verbs have a very specific meaning of including an extra arg param, and it may get really confusing if they don't, so I appreciate fixing that.